Makefile

```
@@ -63,52 +63,57 @@ lint:
 unit-test:
 	go test -cover -coverprofile=coverage.txt ./...
 
-### e2e-test:             Run e2e test cases (kind is required)
+### e2e-test:             Run e2e test cases (in existing clusters directly)
 .PHONY: e2e-test
-e2e-test: ginkgo-check push-images-to-kind
+e2e-test: ginkgo-check push-images
 	kubectl apply -k $(PWD)/samples/deploy/crd
 	cd test/e2e \
 		&& go mod download \
+		&& export REGISTRY=$(REGISTRY)
```

```suggestion
		&& export REGISTRY=$(REGISTRY) \
```

need to be modified here.

---

Done